tools/pyorac/definitions.py

```diff
@@ -549,10 +549,11 @@ def sad_dir(self, sad_dirs, inst):
             if "AVHRR" in inst.sensor:
                 fdr_name = join(fdr, inst.sensor.lower() + "-" +
                                 inst.noaa + "_" + self.sad)
+            if len(self.name) == 3:
+                file_name = "_".join((inst.inst, self.name, "RBD", "Ch*.sad"))
             else:
-                fdr_name = join(fdr, inst.sensor.lower() + "_" + self.sad)
-
-            file_name = "_".join((inst.sensor + "*", self.name, "RBD", "Ch*.sad"))
+                file_name = "_".join((inst.platform.lower(), inst.sensor.lower(),
```

Comment:
I've added a method to the `ParticleType` class that deals with the netcdf filenames. See here: https://github.com/andyprata/orac/blob/5643ce814e441d5c33c468d046b4af625dbfa8d5/tools/pyorac/definitions.py#L568